mappings/net/minecraft/recipe/RecipeUnlocker.mapping

```diff
@@ -1 +1,5 @@
 CLASS net/minecraft/unmapped/C_dscbrwbj net/minecraft/recipe/RecipeUnlocker
+	FIELD f_grnqamso id Lnet/minecraft/unmapped/C_ncpywfca;
+	FIELD f_iowwhwgx recipe Lnet/minecraft/unmapped/C_awrmdwqd;
+	METHOD m_pgxxogft getId ()Lnet/minecraft/unmapped/C_ncpywfca;
+	METHOD m_ybggkzhm getRecipe ()Lnet/minecraft/unmapped/C_awrmdwqd;
```

Comment:
```suggestion
	METHOD m_pgxxogft id ()Lnet/minecraft/unmapped/C_ncpywfca;
	METHOD m_ybggkzhm recipe ()Lnet/minecraft/unmapped/C_awrmdwqd;
```
it's a `record`

Reply:
Ah, do records not use `get`?

Reply:
Yeah, see https://github.com/QuiltMC/quilt-mappings/pull/478#discussion_r1295230021 (I didn't realize it was a record and suggested adding 'get')
It's because if you create a record the normal way (not the way seen here from decompiled code), the field name always matches the getter name.